test/CMakeLists.txt

```diff
@@ -2,16 +2,14 @@ add_subdirectory(src)
 
 enable_testing()
 add_definitions(-DREPO_API_LIBRARY)
-
+# override as we do not want to impose license checks on the test project
```

Comment:
I don't think this is necessary. By default it's `off`.

The user would have to make a conscious decision to turn it on, in which case it should be tested within the test case.

Reply:
can do this, but I have not confirmed if the tests pass with the licensing enabled.